scripts/vbox-resize-disk1.sh

```diff
@@ -0,0 +1,24 @@
+set -ex
```

Comment:
Can you rename the scripts directory to `contrib`? Also, I think this script needs to be made executable.

Reply:
done.